darwin/future/meta/queries/item.py

```diff
@@ -143,3 +144,25 @@ def archive(self) -> None:
         ids = [item.id for item in self]
         filters = {"item_ids": [str(item) for item in ids]}
         archive_list_of_items(self.client, team_slug, dataset_ids, filters)
+
+    def set_layout(self, layout: ItemLayout) -> None:
+        if "team_slug" not in self.meta_params:
```

Comment:
I'd maybe space out these individual `if`s a bit, but that's a minor point!

Reply:
...actually a bit surprised that Black didn't tbh

Reply:
Good point, good for readability